README.md

```diff
@@ -87,6 +87,7 @@
 
 ## Services
 
+- [Forward Email](https://forwardemail.net) (@forwardemail) - the 100% open-source and privacy-focused email service.
```

Comment:
Please remove @forwardemail and the added item needs to be at the end of the Services list.

Thanks.